build-data/generated/entity_map.json

```diff
@@ -128152,7 +128077,8 @@
             "RACKSPACE"
         ],
         "properties": [
-            "spaceweather.com"
+            "spaceweather.com",
+            "spaceweathergallery2.com"
```

Comment:
@GuiltyDolphin mind updating this entity? I think RACKSPACE was a bad ID that must have come from either SSL certs or whois data

Reply:
Have updated it to author name